src/scores/continuous.py

```diff
@@ -1,36 +1,48 @@
 """
 This module contains methods which may be used for continuous scoring
 """
+from typing import Tuple, Union
+
+import pandas as pd
+import xarray as xr
 
 import scores.utils
 
+OBS_TYPE = Union[xr.Dataset, xr.DataArray, pd.DataFrame, pd.Series]  # pylint: disable=invalid-name
 
-def mse(fcst, obs, reduce_dims=None, preserve_dims=None, weights=None):
-    """
 
-    Returns:
-      - By default an xarray containing a single floating point number representing the mean absolute
-        error for the supplied data. All dimensions will be reduced.
-      - Otherwise: Returns an xarray representing the mean squared error, reduced along
-      the relevant dimensions and weighted appropriately.
+def mse(
+    fcst: OBS_TYPE, obs: OBS_TYPE, reduce_dims: Tuple[str] = None, preserve_dims: Tuple[str] = None, weights=None
+) -> Union[xr.Dataset, xr.DataArray]:
```

Comment:
The doc for the return type doesn't match this type hint. I think the types specified in the docstring are what's wanted, which I think is the same as what's in OBS_TYPE?

Reply:
I have reset and rebased the commits, all additional type hints previously in this MR have been removed.